modules/gapi/perf/common/gapi_imgproc_perf_tests.hpp

```diff
@@ -32,6 +32,13 @@ class DilatePerfTest : public TestPerfParams<tuple<compare_f, MatType,int,cv::Si
 class Dilate3x3PerfTest : public TestPerfParams<tuple<compare_f, MatType,cv::Size,int, cv::GCompileArgs>> {};
 class SobelPerfTest : public TestPerfParams<tuple<compare_f, MatType,int,cv::Size,int,int,int, cv::GCompileArgs>> {};
 class SobelXYPerfTest : public TestPerfParams<tuple<compare_f, MatType,int,cv::Size,int,int, cv::GCompileArgs>> {};
+class LaplacianPerfTest : public TestPerfParams<tuple<compare_f, MatType,int,
+                                                      cv::Size,int,
```

Comment:
mb you can carry the inheritance on the next line to polish the appearance?

Reply:
Mb. What do you think about "global alignment"?

Reply:
I don't really know.. Now your new lines are too long, as you can see. I'd prefer you to realign a little, but, honestly, it's not so important. Generally speaking, your global alignment looks good)